src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class BulldozerProducer
+        implements MachineProducer {
```

Comment:
```suggestion
        implements MachineProducer<Bulldozer> {
```
what do you think? same for all child producers